### PR TITLE
fix(ci): msg-system deploy fix jobs for stable and dev

### PR DESCRIPTION
**Pull request by vdovhanych**
Fixes the deployment of msg-system config and adds cache invalidation to the job. 

Will squash the pr into one commit before merging. But first, I wanted to ask if it's ok to do it like this or we should have a simple bash script we would execute and not have the commands directly in the gitlabci file?

### CODE REVIEW

File: ci/deploy.yml [hunk 1]
```diff
@@ -79,7 +79,9 @@ msg-system deploy dev:
   tags:
     - deploy
   script:
+    - source ${MSG_SYSTEM_DATA_DEPLOY_KEYFILE}
     - aws s3 cp packages/suite-data/files/message-system/config.v1.jws s3://data.trezor.io/config/develop/config.v1.jws
+    - aws cloudfront create-invalidation --distribution-id E1ERY5K2OTKKI1 --paths '/config/develop/*'
```
Comment (tsusanka) [inline ci/deploy.yml]: Can we extrapolate the distribution id to a variable just for a better clarity?

Comment (vdovhanych) [reply ci/deploy.yml]: Yes, that's better. What about having the script which we would run in this job? Or it's unnecessary over-complication? 
ad variable: `AWS_CLOUDFRONT_DATA_DID` ?

Comment (tsusanka) [reply ci/deploy.yml]: Maybe just AWS_CLOUDFRONT_ID?

**Comment by tsusanka** [comment]
> But first, I wanted to ask if it's ok to do it like this or we should have a simple bash script we would execute and not have the commands directly in the gitlabci file?

Yes this is fine. I don't think it makes sense to have a bash script with two commands.

So this LGTM, let's squash and force push?

**Comment by tsusanka** [comment]
As @vdovhanych is on vacation, let me finish this one.

**Review by tsusanka — APPROVED** [review]

**Pull request merged**